2022-01/2022-01-20-EmployeeCompromise.md

```diff
@@ -0,0 +1,54 @@
+# Suspected Chia Employee Compromise
+
+## Issue Summary
+
+__Type of Incident Detected__: Potential compromise of employee's Google Account
+
+__Initial Concerns__: On Thursday, January 20th, 2021 at 22:04 (UTC) Employee's account was suspended by an admin account, per the audit logs. 
+                      However, the admin did not perform the suspension of the account and the action originated from an IP address associated with an
+                      AWS EC2 instance.
+
+__Involved Parties__: 
+Affected employee - Individual whose account was suspended.
+
+Admin employee: Individual whose account performed the suspension action.
+
+IPS team: Infrastructure, Platforms, and Security members who responded to the potential incident.
+
+## Timeline (all times UTC )
+
+Date/Time Stamps (Captured from keybase messages)
+
+| Day | Date (2021) | Time (UTC) | Event |
+|-----|-------------|-----------|-------|
+| Thur | Jan 20 | 22:40 | Affected employee logged out of Google Workspace account and received a prompt that his account was disabled, upon attempting to log back in.
+| Thur | Jan 20 | 22:06 | IPS team disables users Okta accounts and searched for abnormal activity. No abnormal login activity for affected employees.
```

Comment:
Affected employee's okta acount was not disabled, only Admin employee

Reply:
I intended that for admin employee, but substituted user. Its fixed now.